pkgs/applications/window-managers/openbox/default.nix

```diff
@@ -7,9 +7,11 @@ stdenv.mkDerivation rec {
   version = "3.6.1";
 
   nativeBuildInputs = [
+    autoreconfHook
     pkgconfig
     makeWrapper
-    python2.pkgs.wrapPython
+    python3.pkgs.wrapPython
+    python3
```

Comment:
I believe, in this case, `python3` belongs to `buildInputs`.

Reply:
Hm, not sure I understand this, but fixed nonetheless.